pkg/exchange/binance/stream.go

```diff
@@ -41,6 +52,7 @@ type Stream struct {
 
 	client        *binance.Client
 	futuresClient *futures.Client
+	privateKey    ed25519.PrivateKey
```

Comment:
you can read the privateKey field from the given exchange object directly since it's in the same package

Reply:
you can also add an accessor like "getEd25519PrivateKey" method on the exchange struct

Reply:
Done.